test/metric/query-json.json

```diff
@@ -19,6 +19,6 @@
       }
     }
   ],
-  "StartTime": "2022-10-03T15:24:0000",
-  "EndTime": "2022-12-5T15:37:0000"
+  "StartTime": "2023-2-11T15:24:0000",
```

Comment:
Why does this need to change. Will it need to change again in the future?

Reply:
Its used to validate the metrics are in CloudWatchMetrics by using query-json.json. There are no harm of changing it

Reply:
Why do we need to change it? If it's for manual validation, we should take out these values so we can script it.

Reply:
I'm good with not changing it. IMO, it would be better to add these flags into the script since developers need to know whether the flag exist but if added in the scripts, we only need to fix the time. It would be reduce time though.